app/lib/constants/site_config.rb

```diff
@@ -5,14 +5,18 @@ module SiteConfig
         description: "How can users sign in?",
         placeholder: ""
       },
-      campaign_hero_html_variant_name: {
-        description: "Hero HtmlVariant name",
-        placeholder: ""
-      },
       campaign_articles_require_approval: {
         description: "",
         placeholder: "Campaign stories show up on sidebar with approval?"
       },
+      campaign_featured_tags: {
+        description: "Posts with which tags will be featured in the campaign sidebar (comma separated, letters only)",
```

Comment:
what do you think about `Posts with these tags` or `Tags whose posts...`

Reply:
@rhymes this sounds like better grammar. My next pass is going to fix the language and make this more descriptive and concise for new communities. I'll tackle it then. Thanks 😊